src/exp2.jl

```diff
@@ -0,0 +1,66 @@
+# Import the generated code
+for i=1:15
+    include("exp_generated/exp_$i.jl")
+end
+
+function getmem(cache,k)
+    return cache[k-1];
+end
+
+function _exp2!(A; caches=nothing, do_balancing = A isa StridedMatrix)
+    n = LinearAlgebra.checksquare(A)
+    nA = opnorm(A,1);
+
+    # Maybe to balancing
+    if do_balancing
+        ilo, ihi, scale = LAPACK.gebal!('B', A)    # modifies A
+    end
+
+    # Select how many multiplications to use
+    rhov=    [0.015; 0.25; 0.95; 2.1; 5.4];
```

Comment:
This could be moved outside and become a global constant, so that Julia won't allocate it each time when this function is invoked.